beacon-chain/rpc/service.go

```diff
@@ -261,13 +311,23 @@ func (s *Service) Stop() error {
 		s.grpcServer.GracefulStop()
 		log.Debug("Initiated graceful stop of gRPC server")
 	}
+	if s.slasherConn != nil {
+		s.slasherConn.Close()
+	}
 	return nil
 }
 
 // Status returns nil or credentialError
 func (s *Service) Status() error {
+	var err error
 	if s.credentialError != nil {
-		return s.credentialError
+		err = s.credentialError
```

Comment:
I dont understand why not return the error here?

Reply:
if there are two credentials errors the user can resolve both of them

Reply:
changed back